README.md

```diff
@@ -11,14 +11,18 @@ This is built on [React.js](https://reactjs.org/) and is deployed on Vercel
 ### How do I contribute?
 
 1. Join the [#project_codesydneywebsite Slack channel](https://codesydney.slack.com/archives/C030XRCQ5A5) and start contributing
-2. We have open issues that can be worked on in the [Issues tab](https://github.com/codesydney/codesydneywebsite/issues). You can see how these issues are progressing on our [Kanban board](https://github.com/codesydney/codesydneywebsite/projects/1). 
-3. Visit our [Code.Sydney Playbook](https://github.com/codesydney/code-sydney-playbook/blob/main/docs/pull-request.md) here for a full guide 
+2. Join the [#project_codesydneywebsite Discord channel](https://discordapp.com/channels/1041795715757789204/1043385733085413396)
+3. We have open issues that can be worked on in the [Issues tab](https://github.com/codesydney/codesydneywebsite/issues). You can see how these issues are progressing on our [Kanban board](https://github.com/codesydney/codesydneywebsite/projects/1).
+4. Visit our [Code.Sydney Playbook](https://github.com/codesydney/code-sydney-playbook/blob/main/docs/pull-request.md) here for a full guide
 
 ### Setting up your machine for local development
 
-1. Clone this to your local machine `git clone git@github.com:codesydney/codesydneywebsite.git`
-2. `npm install`
-3. `npm start`
-4. Access the site using your browser of choice via http://localhost:3000/
+1. If you need any help to fork this repository (https://github.com/firstcontributions/first-contributions)
+2. Clone this to your local machine `git clone git@github.com:codesydney/codesydneywebsite.git` you can copy your own code url after fork
```

Comment:
It might be better to remove the link and tell the contributors to clone from their fork. Not your fault because the line was already there.

I can see a user cloning from `codesydney/codesydneywebsite` and going what the hell? I can't push my changes there.

Can you please help me out with this one and remove the link?